config/default.js

```diff
@@ -52,5 +52,8 @@ module.exports = {
   MEMBERSHIP_TYPES: {
     Group: 'group',
     User: 'user'
-  }
+  },
+
+  REDIS_URL: process.env.REDIS_URL || 'redis://localhost:6379',
```

Comment:
This should not default to localhost. Also it should be asserted,
assert(REDIS_URL,"REDIS_URL is not defined")

Reply:
updated this, but as we are doing for all other services, if the REDIS_URL is not set in the ENV then we are falling back to the localhost